doc/first-steps-with-spyder.rst

```diff
@@ -51,12 +102,39 @@ Customization
 
 Learn how to customize Spyder’s interface to match your workflow and development style.
 
+* Choose your preferred fonts 
+* Switch between different interface, icon and syntax themes 
+* Show, hide, undock and rearrange  Spyder panes 
+* Split, close and pop out Editor panels
+
 .. youtube:: -dARZBUDk_s
             :height: 360
             :width: 640
             :align: left
 
-* Choose your preferred fonts 
-* Switch between different interface, icon and syntax themes 
-* Show, hide, undock and rearrange  Spyder panes 
-* Split, close and pop out Editor panels
+.. rst-class:: dropdown-videos
+
+.. dropdown:: Transcript
+   :animate: fade-in
+
+   .. div:: scroll
+
+      Hello everyone, I'm Juanita! In this video, I will show you how to customize Spyder to match your workflow and development style.
+ 
+      First, we are going to learn how to change the font in the Editor, IPython Console and Help panes. To do this, go to Preferences, select the Appearance entry and scroll down to Fonts. You can change both the style and the size of the font for both plain and rich text. You can see how this affects the font in the Editor, Console and Help panes.
+
+      In this same dialog, you can easily change the syntax highlighting theme, for which you can see the preview at the right of the window. Note that the Spyder's interface theme changes to match the highlighting theme because the Interface theme option is set to Automatic by default. However, you can change the theme for the entire Spyder interface choosing between Light and dark. After selecting this change, click Apply to restart Spyder to apply the new theme.
```

Comment:
```suggestion
      In this same dialog, you can easily change the syntax highlighting theme, for which you can see the preview at the right of the window. Note that Spyder's interface theme changes to match the highlighting theme because the Interface theme option is set to Automatic by default. However, you can change the theme for the entire Spyder interface, choosing between Light and Dark. After selecting this change, click Apply to restart Spyder to apply the new theme.
```

Could have sworn I fixed this...